bindings/c/test/mako/mako.h

```diff
@@ -153,6 +164,9 @@ typedef struct {
 /* args for threads */
 typedef struct {
 	int thread_id;
+	int elem_size[MAX_OP];
```

Comment:
IIUC, this `elem_size[*]` is always a multiple of `LAT_BLOCK_SIZE`. If so, can we add a comment for that?